compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2337,6 +2325,76 @@ OMR::Z::TreeEvaluator::lbitpermuteEvaluator(TR::Node *node, TR::CodeGenerator *c
    return TR::TreeEvaluator::bitpermuteEvaluator(node, cg);
    }
 
+TR::Register*
```

Comment:
Can we use `inlineVectorUnaryOp` ? It does most of the work this two function does.